Misc/NEWS.d/3.9.0a1.rst

```diff
@@ -3117,9 +3117,9 @@ Ensure cookies with ``expires`` attribute are handled in
 .. section: Library
 
 Fix an unintended ValueError from :func:`subprocess.run` when checking for
-conflicting `input` and `stdin` or `capture_output` and `stdout` or `stderr`
-args when they were explicitly provided but with `None` values within a
-passed in `**kwargs` dict rather than as passed directly by name. Patch
+conflicting ``input`` and ``stdin`` or ``capture_output`` and ``stdout`` or ``stderr``
```

Comment:
maybe these should be italicised, since we're talking about parameters being passed?

```suggestion
conflicting *input* and *stdin* or *capture_output* and *stdout* or *stderr*
```